src/GameLogic/Campaign.ts

```diff
@@ -0,0 +1,20 @@
+import {Level} from './Level';
+import {CampaignMetadata} from './Store/CampaignMetadata';
+
+export class Campaign {
+	public readonly id: string;
+
+	public name: string;
+
+	public levels: Level[];
```

Comment:
I'm probably jumping ahead here (so feel free to ignore this comment) but I assume at some point we will also have a way of storing which levels have been completed? Would that be something stored in campaigns, or is it more a property of the level itself?

Reply:
Actually I think a better way would be a separate store rather than holding the info inside a campaign/level, for a couple of reasons:
1. If we decide to allow multiple player profiles in a single game (like in DROD) then it's easier to have them.
2. If the campaign/level has information about the save, each time you want to store new progress you have to serialize the whole campaign and update the record in the store - pretty resource heavy operation.
3. General separation of concerns

So I think a better option will be to have saves stored completely separately.